index.js

```diff
@@ -8,8 +8,16 @@ module.exports = {
   init: function() {
     this._super.init && this._super.init.apply(this, arguments);
 
-    const reporter = new Reporter();
-    const dependencyChecker = new DependencyChecker(this.project, reporter);
-    dependencyChecker.checkDependencies();
+    // When running `ember <command>`, find the `<command>`
+    const emberPosition = process.argv.findIndex(arg => arg.endsWith('/ember'));
+    const ranWithInit = process.argv[emberPosition + 1] === 'init';
+
+    if (ranWithInit) {
+      process.emitWarning('Skipped dependency checker…');
```

Comment:
Hi! Was wondering if this is actually needed. Will this log a warning message in the user's terminal? If so, I think it could be confusing for people new to Ember?

Reply:
That’s a good question, could be! However, could new people to Ember also find it confusing that dependency check happens on every command except `init`, without any warning nor explanation?

Reply:
I could be wrong, but I think (most) people (new to Ember) don't know dependency checking happens when running an Ember command, unless it fails. Skipping silently here seems better to me than emitting a warning. There's also nothing the user can do to address it I think.

Reply:
Yup, there’s definitely some value to what you said. 👍 Maybe someone else on this PR can chime in…?

Reply:
I think `dependency-checker` should be silent if the things are working as expected. So we can remove the log.